.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: Bug Report - MauiCompat
+    url: https://github.com/xamarin/XamarinCommunityToolkit/issues/new?assignees=&labels=bug%2C+unverified%2C+maui-compat&template=bug_report_template.md&title=%5BBug%5D+%5BMauiCompat%5D+
```

Comment:
Sadly it seems that just referring to a template file in the url does not apply labels or the title

Reply:
No problem, I wanted to highlight this as it is now duplicated, the labels are defined in the template file on the XCT repo and also here. I doubt this is likely to change though.